s3tests_boto3/functional/test_s3.py

```diff
@@ -9410,14 +9424,16 @@ def test_lifecycle_expiration_noncur_tags1():
     # noncurrent version expiration at 4 "days"
     response = setup_lifecycle_noncur_tags(client, bucket_name, 4)
 
+    lc_interval = get_lc_debug_interval()
+
     num_objs = verify_lifecycle_expiration_noncur_tags(
-        client, bucket_name, 20)
+        client, bucket_name, 2*lc_interval)
 
     # at T+20, 10 objects should exist
     eq(num_objs, 10)
 
     num_objs = verify_lifecycle_expiration_noncur_tags(
-        client, bucket_name, 40)
+        client, bucket_name, 5*lc_interval)
```

Comment:
My intuitions from prior experimentation was that assuming LC can schedule and expire in less than 1 debug interval, with interval 10s, is not always safe.

Reply:
I have adjusted the sleep interval to >=T+2 (T-debug interval) to account for any latency and also  increased the days count between successive lifecycle rules to accommodate that extra time. So far the tests have passed. 

I will trigger the test run few more times to confirm the same.